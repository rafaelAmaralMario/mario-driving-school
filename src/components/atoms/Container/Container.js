import PropTypes from "prop-types";
import styled from 'styled-components';

import { breakAt, breakPointsSize } from "styles/BreakPoints";

const Container = styled.div`
  width: 100%;
  padding: 0 8px;

  ${breakAt(breakPointsSize.xs)} {
    padding: 0 16px;
  }

  ${breakAt(breakPointsSize.lg)} {
    max-width: 1140px;
    margin: 0 auto;
    padding: 0;
  }
`;


Container.defaultProps = {
  children: undefined
}

Container.propTypes = {
  children: PropTypes.node
}

export default Container;
