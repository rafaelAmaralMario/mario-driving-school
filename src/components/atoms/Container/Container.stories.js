import React from "react";
import Container from "./Container";

export default {
  title: "Components|Atoms/Container",
  component: Container,
};

export const usage = () => (
  <Container></Container>

);
