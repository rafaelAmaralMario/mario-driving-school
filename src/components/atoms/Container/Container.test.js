import React from 'react'
import { render } from 'utils/test-utils';

import Container from './Container';

test('should match with snapshot', () => {
  const { asFragment } = render(<Container></Container>);
  expect(asFragment()).toMatchSnapshot();
})
