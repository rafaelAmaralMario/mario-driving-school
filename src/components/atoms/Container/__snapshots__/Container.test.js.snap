// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match with snapshot 1`] = `
<DocumentFragment>
  .c0 {
  width: 100%;
  padding: 0 8px;
}

@media (min-width:undefinedpx) {
  .c0 {
    padding: 0 16px;
  }
}

@media (min-width:992px) {
  .c0 {
    max-width: 1140px;
    margin: 0 auto;
    padding: 0;
  }
}

<div
    class="c0"
  />
</DocumentFragment>
`;
